core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/signup/SignupEnterUsernamePasswordViewModel.kt

```diff
@@ -272,6 +274,10 @@ class SignupEnterUsernamePasswordViewModel(
 
                 }
 
+                savePasswordUseCase?.invoke(
+                    username =savePerson.username.toString(),
+                    password = _uiState.value.password.toString())
```

Comment:
Indentation - final closing bracket should be on a new line as per Kotlin stlye e.g.
```
savePasswordUseCase?.invoke(
    username =savePerson.username.toString(),
    password = _uiState.value.password.toString()
)
```